core/state/statedb.go

```diff
@@ -233,10 +233,11 @@ func (self *StateDB) GetCodeHash(addr common.Address) common.Hash {
 
 func (self *StateDB) GetState(a common.Address, b common.Hash) common.Hash {
```

Comment:
The idiomatic way of doing this in Go is by naming the return variables. Like,

```
func (self *StateDB) GetState(a common.Address, b common.Hash) (r common.Hash) {
    // ...
    return
}
```

That way you don't have to duplicate the declaration.